pkg/plugins/golang/v4/scaffolds/internal/templates/test/utils/utils.go

```diff
@@ -91,76 +81,99 @@ func Run(cmd *exec.Cmd) ([]byte, error) {
 	return output, nil
 }
 
-// UninstallPrometheusOperator uninstalls the prometheus
-func UninstallPrometheusOperator() {
-	url := fmt.Sprintf(prometheusOperatorURL, prometheusOperatorVersion)
-	cmd := exec.Command("kubectl", "delete", "-f", url)
-	if _, err := Run(cmd); err != nil {
-		warnError(err)
+// GetProjectDir will return the directory where the project is
+func GetProjectDir() (string, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return wd, err
+	}
+	wd = strings.Replace(wd, "/test/e2e", "", -1)
+	return wd, nil
+}
+
+// GetConfig retrieves the Kubernetes configuration file.
+func GetConfig() (*rest.Config, error) {
```

Comment:
Maybe we can use some already existing function for this job? I use this in one project https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client/config#GetConfigOrDie

Reply:
Thank you for your suggestion, I've applied it. :)